libs/watt/src/lib/components/table/watt-table.component.ts

```diff
@@ -291,7 +286,7 @@ export class WattTableComponent<T> implements OnChanges, AfterViewInit, OnDestro
   _element = inject<ElementRef<HTMLElement>>(ElementRef);
 
   /** @ignore */
-  _subscription!: Subscription;
+  private destroy$ = new Subject<void>();
```

Comment:
Suggestion: use `takeUntilDestroyed ` instead :) 
https://indepth.dev/posts/1518/takeuntildestroy-in-angular-v16#takeuntildestroy